Makefile

```diff
@@ -6,8 +6,8 @@ WORKSPACES_WITH_RUST_MODULES := $(wildcard $(addsuffix /crates, $(wildcard ./pac
 # Gets all individual crates such that they can get built
 CRATES := $(foreach crate,${WORKSPACES_WITH_RUST_MODULES},$(dir $(wildcard $(crate)/*/Cargo.toml)))
 
-# add local Cargo install path and use it as custom shell PATH
-PATH := ${PATH}:${CURDIR}/.cargo/bin
+# add local Cargo install path and use it as custom shell PATH (only once)
+PATH := $(PATH::${CURDIR}/.cargo/bin=):${CURDIR}/.cargo/bin
```

Comment:
I've tried to find the documentation to understand the sentence `(only once)` bt didn't find it. Is this the expected behaviour?

Reply:
We started executing make recursively. Without this change the cargo path would be appended to`PATH` multiple times which can lead to issues down the road. This change makes sure any previously appended one is removed first.